doc/man/zcash-tx.1

```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.13.
-.TH ZCASH-TX "1" "July 2020" "zcash-tx v3.1.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.16.
+.TH ZCASH-TX "1" "August 2020" "zcash-tx v4.0.0-rc1" "User Commands"
 .SH NAME
-zcash-tx \- manual page for zcash-tx v3.1.0
+zcash-tx \- manual page for zcash-tx v4.0.0-rc1
```

Comment:
It looks like either the first `-` shouldn't be escaped or the second `-` should be.

Reply:
This is just what the script outputs and so it's always been like this (for example, [see 3.1.0-rc1](https://github.com/zcash/zcash/commit/e255e8bd0a79326fc1827574a85448bd9e9f8b51#diff-3020b72dc7023f5e79b1ca6b5c03ad1dR4)), I think you're right though. I opened an issue: #4689